app/controllers/requests_controller.rb

```diff
@@ -128,9 +128,9 @@ def filter_param
 
   def filtered_requests
     if @filter == :planned
-      Request.planned.reorder(schedule_send_for: :desc).include_associations
+      Request.planned.reorder(schedule_send_for: :desc).includes(:tags)
     else
-      Request.broadcasted.include_associations
+      Request.broadcasted.includes(:tags)
```

Comment:
On the request index page, we don't use anything that needs to be eager loaded other than the tags.